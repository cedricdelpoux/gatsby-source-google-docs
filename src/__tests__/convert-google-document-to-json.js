const data = require(`./data.json`)

const convertGoogleDocumentToJson = require("../convert-google-document-to-json")
const convertJsonToMarkdown = require("../convert-json-to-markdown")

test("converts document", () => {
  const content = convertGoogleDocumentToJson(data)
  const metadata = {id: `hi`, metadatIsCool: `yeah`}
  expect(convertJsonToMarkdown({content, metadata})).toMatchSnapshot()
})
