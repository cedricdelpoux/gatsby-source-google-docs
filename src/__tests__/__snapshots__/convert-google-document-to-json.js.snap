// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`converts document 1`] = `
"---
id: hi
metadatIsCool: yeah
---

## headers

# Header 1

## Header 2

### Header 3

#### Header 4

##### Header 5

## Indented text


Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aliquam hendrerit mi posuere lectus. Vestibulum enim wisi, viverra nec, fringilla in, laoreet vitae, risus.




## Code


Markdown.generate();

## Lists


 1. Red
 2. Green
 3. Blue





 1. Buy flour and salt
 2. Mix together with water
 3. Bake 

## Links


This is [an example](http://example.com/) link.

## Inline formatting


Italics and bold and bold italics and some underlined text and striked out 


This paragraph has some code in it.

## Images

![](https://lh3.googleusercontent.com/Z4q7bVeLnbi4CpZYOCMAOU4_H0uovwNvElapqBEC1DVp93TXFVpyFtCZG6xj_j3l85XsTsj7zDxpoiqWDZj0-DmIUeBAazlC2a59Z7uJqPWrwDegGxBt_pKp5Qj2yeYgVSxGz-61sBmuP7kfxw)



"
`;
